### PR TITLE
JBDS-4059 make...

### PR DESCRIPTION
**Pull request by nickboldt**
JBDS-4059 make org.jboss.tools.richfaces.feature no longer depend on org.jboss.tools.vpe.feature and org.jboss.tools.jsf.feature

### CODE REVIEW

**Comment by alexeykazakov** [comment]
Looks good. But please investigate why all tests failed.

**Comment by nickboldt** [comment]
testPR

**Comment by nickboldt** [comment]
PR build failed because of 'Caused by: java.lang.OutOfMemoryError: Java heap space'
https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/batch.prcheck/jdk=jdk1.8,label=rhel7/
even though it's got 47 of 47 successful tests run:
https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/batch.prcheck/jdk=jdk1.8,label=rhel7/lastSuccessfulBuild/artifact/jbosstools-integration-tests/tests/org.jboss.tools.batch.ui.bot.test/target/surefire-reports/org.jboss.tools.batch.ui.bot.test.AllTestsSuite.txt

So... I suspect the slave sucks or the job is misconfigured to the test ate too much memory... *shrug*. I didn't set these up so I've no idea how the tests are running or in fact what the tests are doing.

**Comment by alexeykazakov** [comment]
Who set them up? @rhopp?

**Comment by nickboldt** [comment]
well, the Job Ownership panel on https://dev-platform-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/batch.prcheck/ says Ondrej <odockal@redhat.com>.

I would assume similar information for the other 5 jobs.

**Comment by odockal** [comment]
I think that this is caused by too many problems in tests (because tests are not run against devstudio but some eclipse) that are logged in jenkins console (log is 1 GB big and this counts for every test, that is why are we out of memory). It is the very same cause as here: [Jira issue](https://issues.jboss.org/browse/JBDS-4077), after that maybe we will need to restart slaves.

**Comment by nickboldt** [comment]
so, if we can't trust the PR job to correctly review the change, can someone manually vet my suggestions in JBDS-4059 and apply them? I don't have push rights to javaee and vpe repos. @alexeykazakov can you help?

**Comment by alexeykazakov** [comment]
Yes, this is in my todo list for today. I will manually build these PRs and apply them if everything is OK.

File: jsf/features/org.jboss.tools.richfaces.feature/feature.xml [hunk 1]
```diff
@@ -22,9 +22,7 @@
 		<import feature="org.jboss.tools.common.text.ext.feature" version="3.6.0" match="greaterOrEqual" />
 		<import feature="org.jboss.tools.common.ui.feature" version="3.6.0" match="greaterOrEqual"  />
 		<import feature="org.jboss.tools.common.verification.feature" version="3.6.0" match="greaterOrEqual"  />
-		<import feature="org.jboss.tools.vpe.feature" version="3.3.0" match="greaterOrEqual"  />
 		<import feature="org.jboss.tools.jst.feature" version="3.6.0" match="greaterOrEqual"  />
-		<import feature="org.jboss.tools.jsf.feature" version="3.6.0" match="greaterOrEqual"  />
```
Comment (alexeykazakov) [inline jsf/features/org.jboss.tools.richfaces.feature/feature.xml]: Why do you remove JSF feature?

**Comment by nickboldt** [comment]
Because...
https://issues.jboss.org/browse/JBDS-4059?focusedCommentId=13307524&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13307524

**Comment by alexeykazakov** [comment]
See my comment here https://github.com/jbdevstudio/jbdevstudio-product/pull/415#discussion_r83977651

File: jsf/features/org.jboss.tools.richfaces.feature/feature.xml (hunk 1 above)
Comment (nickboldt) [inline jsf/features/org.jboss.tools.richfaces.feature/feature.xml]: I can add it back in now that jsf doesn't depend on vpe.

**Comment by odockal** [comment]
testPR

**Comment by nickboldt** [comment]
Rejected.

**Pull request closed without merging**